auxiliary.py

```diff
@@ -6,6 +6,68 @@
 import serpentTools as st
 
 
+def plot_serpent_axial_collapse(data, name, save, lim, V=1, dire='Z'):
+    '''
+    Plots axial flux from serpent detector.
+    Collapses fluxes from G number of groups to Gp number of groups.
+
+    Parameters:
+    -----------
+    data: [serpenttools format]
```

Comment:
```suggestion
    data: [serpenttools format]
```
Since serpenttools format isn't a python type, maybe add some comments or a link to be more specific. Is it a filename, the file handler, a string representing the file....

Reply:
You are right. I will try to improve the description.